ci/local/build.sh

```diff
@@ -80,7 +80,7 @@ BUILD_SCRIPT=${REPO_PATH_IN_CONTAINER}/ci/gpu/build.sh
 if [ -f \${PREBUILD_SCRIPT} ]; then
     source \${PREBUILD_SCRIPT}
 fi
-yes | source \${BUILD_SCRIPT}
```

Comment:
`yes` seemed to hang the script for me for some reason. Do we need it here?

Reply:
Hmm. This script predates me. I don't see why we'd need it here though. I think it's fine to remove.